axolotl-web/src/components/Message.vue

```diff
@@ -176,16 +184,10 @@ export default {
       return true
     },
     humanifyDate(inputDate){
-      moment.locale(this.$language.current);
```

Comment:
I don't know why, but this doesn't look needed from my side, I removed it and the time displayed are still in French.